### PR TITLE
:memo: Update DNS configuration instructions

### PR DESCRIPTION
**Pull request by trolologuy**
## Why

Closes [#2132](https://github.com/platformsh/platformsh-docs/issues/2132)
Closes [#2561](https://github.com/platformsh/platformsh-docs/issues/2561)

## What's changed

- Reworked the quick-start page to provide "how-to" style step-by-step explanations
- Applied the style guidelines on the page and the subpages

### CODE REVIEW

File: docs/src/domains/quick-start.md [hunk 1]
```diff
@@ -2,38 +2,68 @@
...
 
-{{< /note >}}
+- Have a project that's ready to be live.
+- Have a project plan size that's set to **Standard** or higher.
+- Have a domain and have access to its settings on the registrar's website.
+- (Optional) Have the [CLI](../development/cli/_index.md) installed locally.
+- Make sure that your registrar allows [Canonical Name record (CNAME) on Apex domains or one of the alternatives](/domains/steps/dns.md).
 
-{{< note>}}
+## 1. Configure your DNS provider
```
Comment (CollierCZ) [inline docs/src/domains/quick-start.md:22]: :+1: Good to note the change in order here versus the longer one.

File: docs/src/domains/quick-start.md [hunk 2]
```diff
@@ -62,46 +92,15 @@ highlight=false
...
 
-As soon as you add the domain, Platform.sh no longer serves `main-def456-abc123.eu-2.platformsh.site` at all.
-Instead, `{default}` in `routes.yaml` is replaced with `mysite.com` anywhere it appears when generating routes to respond to.
-
-You can still access the original internal domain by running `platform environment:info edge_hostname -e <BRANCH_NAME>`.
-
 {{< note >}}
-If you are planning on using subdomains across multiple projects, [the setup will differ slightly](./steps/subdomains.md).
+If you are planning on using subdomains across multiple projects, [the setup differs slightly](./steps/subdomains.md).
 {{< /note >}}
```
Comment (CollierCZ) [inline docs/src/domains/quick-start.md:102]: :question: Is this really relevant on a quickstart? It doesn't seem to be part of the main flow. Perhaps it would fit in the What's next section?

File: docs/src/domains/steps/_index.md [hunk 3]
```diff
@@ -3,75 +3,122 @@ title: "Custom Domains - Step by step guide"
...
-The address or CNAME to set for that will vary with the CDN provider.
+The address or CNAME to set for that varies with the CDN provider.
 Refer to their documentation or to the [CDN guide](/domains/cdn/_index.md).
 
-## 2. (Non-CDN version) Configure your DNS provider
+<--->
 
-Configure your DNS provider to point your domain to your Platform.sh production environment domain name.
+---
+title=Without a CDN
```
Comment (CollierCZ) [inline docs/src/domains/steps/_index.md:66]: :hammer_and_wrench: I'd have this tab include step 3 (so `With a CDN` wouldn't have a step 2. That might make it clearer what's necessary with a CDN and without. Though not sure how they appear in the ToC that way :thinking:

File: docs/src/domains/steps/_index.md [hunk 4]
```diff
@@ -80,55 +127,129 @@ The CDN should already have been configured in advance to point to Platform.sh a
...
+   Your router passes the request through to your application which in turn responds with whatever it's supposed to do.
 
-Sometimes it can take Let's Encrypt a couple of minutes to provision the certificate the first time.
-This is normal, and only means the first deploy after enabling a domain may take longer than usual.
-Setting the CNAME record with your DNS provider first helps to minimize that disruption.
+On subsequent requests, your browser knows to connect to `1.2.3.4` for domain `www.mysite.com` and skips the rest.
+The entire process takes only a few milliseconds.
 
-## 4. Bonus steps (Optional)
+## Bonus steps (Optional)
```
Comment (CollierCZ) [inline docs/src/domains/steps/_index.md:193]:
```suggestion
## Optional: Bonus steps
```
Might help to have an intro to this section as well.

File: docs/src/domains/steps/_index.md [hunk 5]
```diff
@@ -80,55 +127,129 @@ The CDN should already have been configured in advance to point to Platform.sh a
...
 
 ### Configure health notifications
 
 While not required, it's strongly recommended that you set up [health notifications](/integrations/notifications.md)
 to advise you if your site is experiencing issues such as running low on disk space.
-Notifications can be sent via email, Slack, or PagerDuty.
```
Comment (CollierCZ) [inline docs/src/domains/steps/_index.md:129]: Not sure how this is relevant to the task at hand, really.

File: docs/src/domains/steps/_index.md [hunk 6]
```diff
@@ -80,55 +127,129 @@ The CDN should already have been configured in advance to point to Platform.sh a
...
 
 ### Configure automatic backups
 
 It's strongly recommended that you set up an [API token](/development/cli/api-tokens.md) and install the CLI
 to define [an automatic backup](/administration/backup-and-restore.md#automated-backups) cron task.
```
Comment (CollierCZ) [inline docs/src/domains/steps/_index.md:203]: Or this

**Review by CollierCZ — CHANGES_REQUESTED** [review]
I started getting caught up in the details, which I shouldn't have, but left in the minor details just cause it's easier than taking them out.

Overall, it seems like the steps are clearer in the new way. But now I'm not really so sure what the point of Quickstart vs. the steps is. It doesn't seem so much quicker this way (once you've made the other part clearer). And now there's a lot of duplication that should be resolved somehow.

Also, I'd say the scope is kind of expanding. Because it seems like a lot of clarity is being added on pages that weren't part of the original issue, while the question in the original issue doesn't seem resolved. is the host of `@` necessary? Does it work the same for Dedicated vs. not?

So I'd say it's good changes, but make sure to focus back on the original issue before the whole thing gets too big.

File: docs/src/domains/steps/_index.md (hunk 4 above)
Comment (trolologuy) [reply docs/src/domains/steps/_index.md]: Since this is the only bonus step, it might be even easier to keep the host file editing.

File: docs/src/domains/quick-start.md [hunk 7]
```diff
@@ -2,38 +2,68 @@
...
-{{< note theme="warning">}}
+This page gives a basic configuration example.
+For more details, see the [Step by step guide](../domains/steps/_index.md).
 
-Adding a custom domain means your existing automatically generated URLs stop working.
+## Before you begin
 
-Don't add a custom domain to your project until you're fully ready to change your DNS settings.
-Until that time, continue working with the Platform.sh generated URLs.
+You need to:
```
[the remark this answers is not in the document]
Comment (trolologuy) [reply docs/src/domains/quick-start.md]: I'm not sure why the periods at the end should be avoided, since it's a bulleted list that's [neither a sequence nor options](https://developers.google.com/style/lists?hl=en#types-of-lists).

File: docs/src/domains/steps/_index.md [hunk 8]
```diff
@@ -3,132 +3,228 @@ title: "Custom Domains - Step by step guide"
...
+configure the domain on your project before switching DNS over.
 {{< /note >}}
 
+## Before you begin
+
+You need to:
+
+- Have a project that’s ready to be live.
+- Have a domain and have access to its settings on the registrar’s website.
+- (Optional) Have the [CLI](/development/cli/_index.md) installed locally.
```
Comment (CollierCZ) [inline docs/src/domains/steps/_index.md:23]: :clipboard: 
```suggestion
- (Optional) Have the [CLI](../../administration/cli/_index.md) installed locally.
```

File: docs/src/domains/steps/_index.md [hunk 9]
```diff
@@ -3,132 +3,228 @@ title: "Custom Domains - Step by step guide"
...
+---
+title=Using the CLI
+file=none
+highlight=false
+---
 
-![Edit Plan](/images/management-console/billing-plan-upgrade.png "0.6")
+1. Sign in to your registrar's management system to configure your domain.
+2. Set the time to live (TTL) on your domain to the lowest possible value to minimize transition time.
+3. Get the CNAME target by running the [CLI](/development/cli/_index.md) command `platform environment:info edge_hostname`.
```
Comment (CollierCZ) [inline docs/src/domains/steps/_index.md:65]: :clipboard:
```suggestion
3. Get the CNAME target by running the [CLI](../../administration/cli/_index.md) command `platform environment:info edge_hostname`.
```

**Review by CollierCZ — CHANGES_REQUESTED** [review]
Will have a more comprehensive review, but this should at least get the build working.

File: docs/src/domains/checklist.md [hunk 10]
```diff
@@ -11,56 +11,54 @@ sidebarTitle: "Pre-Launch Checklist"
...
 the Time-To-Live (TTL) on your domain is set to the lowest possible value in order to minimize transition time.
 
 {{< note >}}
 
-You will not be able to use a `A` record.
-Verify your DNS provider supports CNAMES.
-(If it does not you will want to run away from it anyway).
-Also you will be much happier if it supports Apex domains (more in the next chapter).
+You can't use a `A` record.
+Verify your DNS provider supports CNAME and Apex domains.
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/domains/checklist.md:22]: The :lady_beetle: here wasn't resolved

File: docs/src/domains/steps/subdomains.md [hunk 11]
```diff
@@ -36,27 +35,30 @@ though your DNS records should prevent them from actually using it
...
-browsers would refuse to set a cookie on `example.com` from a page at `foo.example.com`.
-(They would still accept cookies from a page at `example.com`.)
+browsers would refuse to set a cookie on `example.com` from a page at `foo.example.com` but would still accept cookies from a page at `example.com`.
 
 ### Subdomain hijacking protection
 
-By default, Platform.sh allows only one project to use a given domain at a time.
-This is to prevent a malicious actor from registering a project with `evil.example.com`
+By default, the subdomain hijacking protection is activated, allowing only one project to use a given domain at a time.
+This security measure is there to prevent a malicious actor from registering a project with `evil.example.com`
```
Comment (CollierCZ) [inline docs/src/domains/steps/subdomains.md:61]: :question: Do you really think it was unclear before? What might have been misinterpreted? It seems like extra words to express the same thing.

File: docs/src/domains/steps/subdomains.md [hunk 12]
```diff
@@ -65,9 +67,7 @@ that project now owns `example.com` as far as Platform.sh is concerned
 and no other project can have a domain anywhere in `*.example.com`.
 (Multiple subdomains within that same project are perfectly fine.)
 
-In most cases, that's a desirable added layer of security.
```
Comment (CollierCZ) [inline docs/src/domains/steps/subdomains.md:68]: :question: Why take out this sentence? It seemed like a good transition.

File: docs/src/domains/steps/subdomains.md [hunk 13]
```diff
@@ -65,9 +67,7 @@ that project now owns `example.com` as far as Platform.sh is concerned
...
-But you may run into a problem when you want multiple subdomains from the same organization as separate projects.
-(Multiple departments at the same university, for instance.)
+The subdomain hijacking protection can cause issues if you want multiple subdomains from the same organization as separate projects such as for example multiple departments at the same university.
```
Comment (CollierCZ) [inline docs/src/domains/steps/subdomains.md:70]: :hammer_and_wrench: The new version seems longer and more complex without adding much clarity. Unless you have specific concerns about the original that I don't see, the most I could see would be getting rid of the parentheses:
```suggestion
But you may run into a problem when you want multiple subdomains from the same organization as separate projects.
For example, multiple departments at the same university.
```

File: docs/src/domains/steps/subdomains.md [hunk 14]
```diff
@@ -85,7 +85,7 @@ So you could add `beep.example.com` to a different project without any issues.
 
 ## Locked domains
 
-In certain cases (such as if your domain was added manually by Platform.sh support),
+In certain cases (such as if your domain was manually added by the support team),
```
Comment (CollierCZ) [inline docs/src/domains/steps/subdomains.md:88]: :hammer_and_wrench: I feel like this decreases clarity because it removes the idea of what support team it is. I'm also not sure why the word order was changed, since it seems to me to be the wrong emphasis now.
```suggestion
In certain cases (such as if your domain was added manually by Platform.sh support),
```

File: docs/src/domains/steps/subdomains.md [hunk 15]
```diff
@@ -94,12 +94,12 @@ Include the project ID of the project that already has the domain.
...
-If you try to use a domain that's claimed, you see an error like the following: 
+If you try to use a domain that's claimed, you see an error like the following:
 
 ```text
 This domain is already claimed by another project. If this is incorrect or you are trying to add a subdomain, please open a ticket with support.
-``` 
+```
 
-This relates to the [subdomain highjacking prevention](#subdomain-hijacking-protection).
+This relates to the [subdomain hijacking prevention](#subdomain-hijacking-protection).
```
Comment (CollierCZ) [inline docs/src/domains/steps/subdomains.md:103]: :clipboard: 
```suggestion
This relates to the [subdomain hijacking protection](#subdomain-hijacking-protection).
```

File: styles/Vocab/Platform/accept.txt [hunk 16]
```diff
@@ -56,6 +56,7 @@ imagetragick
 jpegoptim
 Lando
 Laravel
+Linux
```
Comment (CollierCZ) [inline styles/Vocab/Platform/accept.txt:59]: :clipboard: This is not a necessary addition.
 ```suggestion
```

File: docs/src/domains/quick-start.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/domains/quick-start.md]: Avoiding end punctuation isn't the goal. The goal is avoiding unnecessary repetition. Avoiding end punctuation is a side effect. The second exception in the section on [end punctuation in lists](https://developers.google.com/style/lists?hl=en#capitalization), which is the relevant section.

File: docs/src/domains/checklist.md [hunk 17]
```diff
@@ -11,56 +11,54 @@ sidebarTitle: "Pre-Launch Checklist"
 ## 1. Register a domain name with a supported provider
 
 You have a domain name registered for your site with a Registrar of your choice.
-The registrar must allow you to use CNAMEs for your domain.
-(Some registrars may call these Aliases or similar.).
+The registrar must allow you to use a Canonical Name record (CNAME) for your domain.
+Some registrars may call these Aliases or similar.
 If your domain is currently active elsewhere,
 the Time-To-Live (TTL) on your domain is set to the lowest possible value in order to minimize transition time.
```
Comment (CollierCZ) [inline docs/src/domains/checklist.md:17]: :clipboard:
```suggestion
the time to live (TTL) on your domain is set to the lowest possible value in order to minimize transition time.
```

**Review by CollierCZ — CHANGES_REQUESTED** [review]
I think it's great that you standardized the steps and overall structure of the articles. That definitely makes it easier to follow.

It also makes it clearer to me that I was right to question whether the division into two pages was necessary. It seems even less so now. Do you see any value coming from having the two pages? The "Quick start" doesn't seem quicker or anything like that, but I could be missing something.

Also, the concerns I raised in my previous review about the original issue not being address remain true. The questions like: Is the host of @ necessary? Does it work the same for Dedicated vs. not? Addressing those would also help.

**Comment by trolologuy** [comment]
Open points / ideas:
1. #2561 -> I think the steps and alternatives should be clearer now. To mitigate, we might want to add a list with links to the most common DNS providers's settings? (Cloudflare, Namecheap, OVHcloud, GoDaddy, ...)
2. Unsure about terminology, would `edge_hostname` make more sense than Target?
3. Do we want to add something like `how to move a domain from project A to project B` somewhere?
4. The [region migration](https://docs.platform.sh/projects/region-migration.html#8-switch-to-the-new-site) part about DNS could be better explained as well.

File: docs/src/domains/steps/tls.md [hunk 18]
```diff
@@ -1,52 +1,38 @@
...
 
-To convert it to the old-style RSA key:
-
-```bash
-openTLS rsa -in private.key -out private.rsa.key
-```
-
-{{< /note >}}
+The use of HA or EV certificates is the main reason why you may wish to use a third-party issuer rather than the default certificate.
+If you use wildcard routes, you need a custom certificate.
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:24]: :hammer_and_wrench: I feel like the order of causation is somewhat confused here in that you can't use a wildcard route until you get a custom certificate, so it's not about you already using it, if that makes sense. Perhaps this?

```suggestion
Another reason you might need a custom certificate is for wildcard routes.
```

Comment (CollierCZ) [reply docs/src/domains/steps/tls.md:24]: Or this, given that this is where I'd expect the detailed information?
```suggestion
Let's Encrypt certificates also don't support wildcard routes,
so you need a custom certificate if you want to use one.
```

File: docs/src/domains/steps/tls.md [hunk 19]
```diff
@@ -1,52 +1,38 @@
...
 
 ### Add a custom certificate
 
-You can add a custom certificate in the [Console](/administration/web/_index.md)
-or using the [command line interface](../../administration/cli/_index.md).
+You can add a custom certificate by using the [command line interface](../../administration/cli/_index.md) or in the [Console](/administration/web/_index.md).
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:28]: :clipboard: 
```suggestion
You can add a custom certificate using the [CLI](../../administration/cli/_index.md) or in the [Console](../../administration/web/_index.md).
```

File: docs/src/domains/steps/tls.md [hunk 20]
```diff
@@ -1,52 +1,38 @@
...
 
-{{< codetabs >}}
+Your certificate has to use the "PKCS#1" format and start with `BEGIN RSA PRIVATE KEY`. If it doesn't, see [how to change the private key format](#how-to-change-the-private-key-format)
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:30]: :clipboard: 
```suggestion
Your certificate has to use the PKCS #1 format and start with `BEGIN RSA PRIVATE KEY`.
If it doesn't, see how to [change the private key format](#how-to-change-the-private-key-format).
```

File: docs/src/domains/steps/tls.md [hunk 21]
```diff
@@ -1,52 +1,38 @@
...
 
----
-title=In the Console
-file=none
-highlight=false
----
+To add your custom certificate:
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:32]: :clipboard: 
```suggestion
To add your custom certificate, follow these steps:
```

File: docs/src/domains/steps/tls.md [hunk 22]
```diff
@@ -55,17 +41,34 @@ highlight=false
...
+</code></pre></div>
 
 For example:
 
 ```bash
 platform domain:add -p abcdefg123456 secure.example.com --cert /etc/TLS/private/secure-example-com.crt --key /etc/TLS/private/secure-example-com.key
 ```
 
-You can optionally include intermediate SSL certificates by adding `--chain <PATH_TO_FILE>` for each one.
+You can optionally include intermediate SSL certificates by adding `--chain` {{<variable "PATH_TO_FILE" >}} for each one.
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:55]: :clipboard: 
```suggestion
You can optionally include intermediate SSL certificates by adding <code>--chain {{<variable "PATH_TO_FILE" >}}</code> for each one.
```

File: docs/src/domains/steps/tls.md [hunk 23]
```diff
@@ -55,17 +41,34 @@ highlight=false
...
+<--->
+
+---
+title=In the Console
+file=none
+highlight=false
+---
+
+1. Open the [Console](https://console.platform.sh)
+2. Click the tile of the project you want to access where you want to add a certificate.
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:66]: :hammer_and_wrench: The context is already given by the tab.
```suggestion
1. Open the project where you want to add a certificate.
```

File: docs/src/domains/steps/tls.md [hunk 24]
```diff
@@ -55,17 +41,34 @@ highlight=false
...
+3. Click {{< icon settings >}} **Settings**.
+4. Click **Certificates**.
+5. Click **+ Add**.
+6. Fill in your private key, public key certificate, and (optionally) intermediate SSL certificates.
+7. Click **Add Certificate**.
 
 {{< /codetabs >}}
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:74]: :lady_beetle: Seems like the part about redeploying (if still true) should be part of the tabs since there's also a way to do it in the console.

File: docs/src/domains/steps/tls.md [hunk 25]
```diff
@@ -75,11 +78,18 @@ For the new certificate to be taken into account, you need to [redeploy the envi
 platform environment:redeploy
 ```
 
-{{< note theme="info" title="Success!" >}}
+Your site is now live and accessible to the world as soon as the DNS propagates.
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:81]: :lady_beetle: That's not what this doc is about. This is about the certificate. It can be live before doing the doc. (Also, DNS doesn't propagate)

File: docs/src/domains/steps/tls.md [hunk 26]
```diff
@@ -75,11 +78,18 @@ For the new certificate to be taken into account, you need to [redeploy the envi
...
 
-Your site should now be live and accessible to the world (as soon as the DNS propagates).
+If something isn't working see the [troubleshooting guide](/domains/troubleshoot.md) for common issues.
+If that doesn't help, [contact support](../../overview/get-support.md).
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:85]: :hammer_and_wrench: I'm not sure these are really relevant to the certificate.
```suggestion
```

File: docs/src/domains/steps/tls.md [hunk 27]
```diff
@@ -75,11 +78,18 @@ For the new certificate to be taken into account, you need to [redeploy the envi
...
 
-{{< /note >}}
+### How to change the private key format
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:86]: :hammer_and_wrench: 
```suggestion
### Change the private key format
```

File: docs/src/domains/steps/tls.md [hunk 28]
```diff
@@ -75,11 +78,18 @@ For the new certificate to be taken into account, you need to [redeploy the envi
...
 
-If something isn't working see the [troubleshooting guide](/domains/troubleshoot.md) for common issues.
-If that doesn't help, feel free to [contact support](../../overview/get-support.md).
+Your certificate's private key has to be in PKCS#1 style, and start with `BEGIN RSA PRIVATE KEY`.
+If it starts with `BEGIN PRIVATE KEY`, it's PKCS#8 and bundled with the identifier for key type.
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:89]: :hammer_and_wrench: 
```suggestion
Your certificate's private key needs to be in PKCS #1 format, which means it has `BEGIN RSA PRIVATE KEY` near the start.
If it has `BEGIN PRIVATE KEY` instead, it's in PKCS #8 format and you need to change it.
```

File: docs/src/domains/steps/tls.md [hunk 29]
```diff
@@ -75,11 +78,18 @@ For the new certificate to be taken into account, you need to [redeploy the envi
...
+
+To convert the RSA key from PKCS#8 to PKCS#1, run:
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:91]: :hammer_and_wrench: 
```suggestion
To convert the key from PKCS #8 to PKCS #1 format, run the following command:
```

File: docs/src/domains/steps/tls.md [hunk 30]
```diff
@@ -75,11 +78,18 @@ For the new certificate to be taken into account, you need to [redeploy the envi
...
+
+```bash
+openTLS rsa -in private.key -out private.rsa.key
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:94]: :question: Not with variables? I'd guess these are the defaults, but are they necessarily followed? Is it necessary to rename the key?

File: docs/src/domains/troubleshoot.md [hunk 31]
```diff
@@ -2,36 +2,36 @@
 title: "Going Live - Troubleshooting"
 weight: 4
 description: |
-  If all steps above have been followed and the site still doesn't resolve (after waiting for the DNS update to propagate), here are a few simple self-help steps to take before contacting support.
+  If your site still doesn't resolve after the DNS changes, check the troubleshooting before contacting support.
```
Comment (CollierCZ) [inline docs/src/domains/troubleshoot.md:5]: :hammer_and_wrench: 
```suggestion
  If your site doesn't resolve after you've made DNS changes, check potential solutions to common issues.
```

File: docs/src/domains/troubleshoot.md [hunk 32]
```diff
@@ -2,36 +2,36 @@
...
 sidebarTitle: "Troubleshooting"
 ---
 
 {{% description %}}
 
 ## Verify DNS
 
-On the command line with OS X or Linux (or using the Linux subsystem for Windows) type `host www.example.com`:
+On the command line using macOS, Linux or the Linux subsystem for Windows, type:
```
Comment (CollierCZ) [inline docs/src/domains/troubleshoot.md:13]: :clipboard: 
```suggestion
On the command line using macOS, Linux, or the Windows Subsystem for Linux, run the following command:
```

File: docs/src/domains/troubleshoot.md [hunk 33]
```diff
@@ -2,36 +2,36 @@
...
+`host www.{{< variable "YOUR_DOMAIN" >}}`.
 
-The response should be something like:
+The response where `www.{{< variable "YOUR_DOMAIN" >}}` is `example.com` should be something like:
```
Comment (CollierCZ) [inline docs/src/domains/troubleshoot.md:16]: :hammer_and_wrench: 
```suggestion
If your domain is `example.com`, the response should be something like the following:
```

File: docs/src/domains/troubleshoot.md [hunk 34]
```diff
@@ -2,36 +2,36 @@
...
 sidebarTitle: "Troubleshooting"
 ---
 
 {{% description %}}
 
 ## Verify DNS
 
-On the command line with OS X or Linux (or using the Linux subsystem for Windows) type `host www.example.com`:
+On the command line using macOS, Linux or the Linux subsystem for Windows, type:
+`host www.{{< variable "YOUR_DOMAIN" >}}`.
```
Comment (CollierCZ) [inline docs/src/domains/troubleshoot.md:14]:
````suggestion

```bash
host www.{{< variable "YOUR_DOMAIN" >}}
```
````

File: docs/src/domains/troubleshoot.md [hunk 35]
```diff
@@ -2,36 +2,36 @@
...
 
-The response should be something like:
+The response where `www.{{< variable "YOUR_DOMAIN" >}}` is `example.com` should be something like:
 
 ```text
-www.example.com is an alias for main-t2xxqeifuhpzg.eu.platform.sh.
-main-t2xxqeifuhpzg.eu.platform.sh has address 54.76.136.188
+www.{{<variable "YOUR_DOMAIN" >}} is an alias for main-def456-abc123.eu-2.platformsh.site.
+main-def456-abc123.eu-2.platformsh.site has address 192.0.2.1
```
Comment (CollierCZ) [inline docs/src/domains/troubleshoot.md:20]: :clipboard: The whole point of having `example.com` above is to use it in the example
```suggestion
www.example.com is an alias for main-abcd123.abcdefgh1234567.eu.platformsh.site.
main-abcd123.abcdefgh1234567.eu.platformsh.site has address 192.0.2.1
```

File: docs/src/domains/troubleshoot.md [hunk 36]
```diff
@@ -2,36 +2,36 @@
...
 
-1. If it isn't, either you haven't configured correctly your DNS server, or the DNS configuration didn't propagate yet.
-   As a first step you can try and remove your local DNS cache.
-2. You can also try to set your DNS server to the Google public DNS server (8.8.8.8/8.8.4.4)
-   to see if the issue is with the DNS server you are using.
-3. Try to run `ping www.example.com` (with you own domain name).
-   If the result is different from what you got from the `host www.example.com`,
-   you might want to verify your `/etc/hosts` file (or its windows equivalent),
-   you might have left there an entry from testing.
+If it isn't:
```
Comment (CollierCZ) [inline docs/src/domains/troubleshoot.md:23]: :hammer_and_wrench: 
```suggestion
If it isn't, try the following steps:
```

File: docs/src/domains/troubleshoot.md [hunk 37]
```diff
@@ -2,36 +2,36 @@
...
+
+* Your DNS server probably isn't correctly configured or the DNS configuration didn't propagate yet. Try and remove your local DNS cache.
```
Comment (CollierCZ) [inline docs/src/domains/troubleshoot.md:25]: :hammer_and_wrench: 
```suggestion
* Your DNS server might not be correctly configured or the old DNS records are still cached.
  Try removing your local DNS cache.
```

File: docs/src/domains/troubleshoot.md [hunk 38]
```diff
@@ -2,36 +2,36 @@
...
+* Set the DNS server of your computer to the Google public DNS server (`8.8.8.8` and `8.8.4.4`) to see if the issue is with the DNS server you are using.
```
Comment (CollierCZ) [inline docs/src/domains/troubleshoot.md:26]:
```suggestion
* Set the your computer's DNS server to Google Public DNS (`8.8.8.8` and `8.8.4.4`)
  to see if the issue is with the DNS server you are using.
```

File: docs/src/domains/troubleshoot.md [hunk 39]
```diff
@@ -2,36 +2,36 @@
...
+* Run `ping www.{{< variable "YOUR_DOMAIN" >}}`.
+   If the result is different from what you got from the `host www.{{< variable "YOUR_DOMAIN" >}}`,
+   IP-overrides could be leftover [from testing](checklist.md#2-test-your-site).
```
Comment (CollierCZ) [inline docs/src/domains/troubleshoot.md:29]:
```suggestion
  If the result is different from what you got from running `host www.{{< variable "YOUR_DOMAIN" >}}`,
  you might need to remove your [test settings](./checklist.md#2-test-your-site).
```

File: docs/src/domains/troubleshoot.md [hunk 40]
```diff
@@ -2,36 +2,36 @@
...
 
 ## Verify SSL
 
-On the command line with OS X or Linux (or using the Linux subsystem for Windows),
-type `curl -I -v  https://example.com` (again using your own domain):
+On the command line using macOS, Linux or the Linux subsystem for Windows,
+type: `curl -I -v  https://www.{{< variable "YOUR_DOMAIN" >}}`.
```
Comment (CollierCZ) [inline docs/src/domains/troubleshoot.md:34]: :clipboard: 
````suggestion
On the command line using macOS, Linux, or the Windows Subsystem for Linux, run the following command:

```bash
curl -I -v  https://www.{{< variable "YOUR_DOMAIN" >}}
```
````

File: docs/src/domains/troubleshoot.md [hunk 41]
```diff
@@ -40,18 +40,16 @@ Often the problem is with a mismatch between the certificate and the domain name
...
 Use a tool such as the [conversion tool provided by Verisign](https://www.verisign.com/en_US/channel-resources/domain-registry-products/idn/idn-conversion-tool/index.xhtml).
 
 ## Something still wrong?
 
 {{% troubleshoot %}}
 
-[Contact support](/overview/get-support.md)
-
-We are here to help. Please include as much detail as possible (we can provide quicker help).
+If your website is still not working as expected, [contact support](/overview/get-support.md)
```
Comment (CollierCZ) [inline docs/src/domains/troubleshoot.md:55]: :clipboard: 
```suggestion
If your website is still not working as expected, [contact support](../overview/get-support.md).
```

File: docs/src/environments/default-environment.md [hunk 42]
```diff
@@ -176,8 +176,8 @@ Follow the instructions to change the default branch to `main` for your provider
 
 ## 7. Update DNS records
 
-Whether or not you're using a CDN,
-if your site is live you have probably added a Platform.sh address somewhere when configuring a [custom domain](../domains/quick-start.md).
+Whether you're using a CDN,
```
Comment (CollierCZ) [inline docs/src/environments/default-environment.md:179]: :clipboard: `Whether` doesn't make sense without an `or not`
```suggestion
Whether or not you're using a CDN,
```

**Review by CollierCZ — CHANGES_REQUESTED** [review]
I think the overall flow is definitely clearer now :+1: . That's a great lift :muscle: 

There are some details to work out. With the rebasing and changes from main, it was hard for me to tell what I should look at and what not. Hopefully, it's clear enough. If anything's unclear, give me a ping.

Also, a page has been deleted, so a redirect should be setup for that page.

**Review by CollierCZ — CHANGES_REQUESTED** [review]
I think the overall flow is definitely clearer now :+1: . That's a great lift :muscle: 

There are some details to work out. With the rebasing and changes from main, it was hard for me to tell what I should look at and what not. Hopefully, it's clear enough. If anything's unclear, give me a ping.

File: docs/src/domains/checklist.md [hunk 43]
```diff
@@ -11,56 +11,54 @@ sidebarTitle: "Pre-Launch Checklist"
 ## 1. Register a domain name with a supported provider
 
 You have a domain name registered for your site with a Registrar of your choice.
-The registrar must allow you to use CNAMEs for your domain.
-(Some registrars may call these Aliases or similar.).
+The registrar must allow you to use a Canonical Name record (CNAME) for your domain.
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/domains/checklist.md]: Would you expect to see HTML written out? Or HTTP on a page like this: https://docs.platform.sh/development/headers.html ?

File: docs/src/domains/steps/_index.md [hunk 44]
```diff
@@ -1,137 +1,186 @@
...
-Refer to their documentation or to the [CDN guide](/domains/cdn/_index.md).
+{{< /codetabs >}}
 
-## 2. (Non-CDN version) Configure your DNS provider
+## 2. Configure your DNS provider
 
-Configure your DNS provider to point your domain to your Platform.sh production environment domain name.
+Your DNS provider (usually your registrar) is where you manage your domain.
+Most registrars offer similar functionalities regarding DNS configuration but use different terminology or configuration.
+For example a few registrars require the use of `@` character to create a CNAME records on the APEX domain where most others registrars don't.
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/domains/steps/_index.md:66]: If this is supposed to answer the problem in the original issue, I'm not so sure it does. The original issue is about a `@` in an alias record, not a CNAME, and it's not clear when it's necessary or not.

File: docs/src/domains/steps/tls.md (hunk 24 above)
Comment (CollierCZ) [reply docs/src/domains/steps/tls.md:81]: Either testing or asking would be good.

File: docs/src/domains/steps/tls.md [hunk 45]
```diff
@@ -1,85 +1,91 @@
...
-{{< /codetabs >}}
+Your certificate's private key needs to be in PKCS #1 format, which means it starts with `-----BEGIN RSA PRIVATE KEY-----`.
+If it has `-----BEGIN PRIVATE KEY-----` instead, it's in PKCS #8 format and you need to change it.
 
-For the new certificate to be taken into account, you need to [redeploy the environment](../../development/troubleshoot.md#force-a-redeploy).
+To convert the private key from PKCS #8 to PKCS #1 format, run the following command:
 
 ```bash
-platform environment:redeploy
+openTLS rsa -in {{<variable "PATH_TO_PRIVATE_KEY_FILE" >}}.key -out {{<variable "PATH_TO_PRIVATE_KEY_FILE" >}}.rsa.key
```
Comment (CollierCZ) [inline docs/src/domains/steps/tls.md:90]: :lady_beetle: Generally, the path to a file includes the extension, so this way of presenting it might be slightly confusing.

File: styles/Vocab/Platform/accept.txt (hunk 16 above)
Comment (CollierCZ) [reply styles/Vocab/Platform/accept.txt:59]: I'm not sure why this was resolved? It still seems unnecessary to me?

File: docs/src/domains/checklist.md [hunk 46]
```diff
@@ -10,69 +10,153 @@ sidebarTitle: "Pre-Launch Checklist"
 
 ## 1. Register a domain name with a supported provider
 
-You have a domain name registered for your site with a Registrar of your choice.
-The registrar must allow you to use CNAMEs for your domain.
-(Some registrars may call these Aliases or similar.).
-If your domain is currently active elsewhere,
-the Time-To-Live (TTL) on your domain is set to the lowest possible value to minimize transition time.
+Before you go live, you need a domain name registered with the registrar of your choice.
+Verify your registrar supports [Canonical Name (`CNAME`) records for apex domains](./steps/dns.md#handling-apex-domains).
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/domains/checklist.md]: I'm afraid I don't understand the question. Could you rephrase?

**Review by CollierCZ — APPROVED** [review]
